cernopendata/modules/fixtures/data/articles/cms-guide-troubleshooting/cms.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "title": "CMS Response to errors",
```

Comment:
All looks good, except for this title, which generates the URL:

- http://0.0.0.0/articles/cms-response-to-errors

with the visible title:

- CMS Response to errors

It may read a bit weird... Do we want to rename the title to something like "CMS Guide Troubleshooting" or something?

Can merge this anyway for quick showcasing to @katilp tomorrow...

Reply:
Ok! I'll add that with the redirection view. Sorry I didn't see it earlier